docs/release/3.4/release-3-4-17.md

```diff
@@ -0,0 +1,321 @@
+OSG Software Release 3.4.17
+===========================
+
+**Release Date**: 2018-08-08
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   OSG 3.4 Repository
+    -   [Singularity 2.6.0](https://github.com/singularityware/singularity/releases/tag/2.6.0): Bug fix release
+        -   This release includes an extra option called "underlay" that is not
+            in the upstream release nor enabled by default but is supported by
+            OSG and recommended to be enabled instead of the "overlay" option.
+            [See the documentation for details.](https://opensciencegrid.org/docs/worker-node/install-singularity/#configuring-singularity)
+    -   [HTCondor 8.6.12](https://www-auth.cs.wisc.edu/lists/htcondor-world/2018/msg00016.shtml): Bug fix release
+        -   Fixed memory leak when SSL authentication fails
+        -   Can now set job environment variables in Singularity containers
+    -   CVMFS X.509 helper: Fixed file descriptor leak
+    -   Gratia probes: Fix for missing Slurm accounting records
+    -   [Pegasus 4.8.3](https://pegasus.isi.edu/2018/08/06/pegasus-4-8-3-released/): Bug fixes and container based example
+    -   [HTCondor-CE 3.1.3](https://github.com/opensciencegrid/htcondor-ce/releases/tag/v3.1.3): Fix for condor\_ce\_info\_status using the wrong port for the central collector
```

Comment:
I think wrapping `condor_ce_info_status` in backticks would look better here

Reply:
Will do.